sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp

```diff
@@ -91,11 +89,12 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Administra
      *
      * @returns Response containing a list of settings.
      */
-    Azure::Response<SettingsListResult> GetSettings(
+    Azure::Response<Models::SettingsListResult>
+    GetSettings(
```

Comment:
This one is probably needed as well - I think clang format will fail if it is 2 lines:
```suggestion
    Azure::Response<Models::SettingsListResult> GetSettings(
```